example/ios/Podfile.lock

```diff
@@ -237,10 +237,10 @@ PODS:
     - React-cxxreact (= 0.63.3)
     - React-jsi (= 0.63.3)
   - React-jsinspector (0.63.3)
-  - react-native-plaid-link-sdk (6.0.2):
-    - Plaid (~> 2.0.3)
+  - react-native-plaid-link-sdk (6.0.4):
```

Comment:
Should we also update the `example/package.json`?
That currently has `6.0.2`

Reply:
:zap: